src/es/components/molecules/multiLevelNavigation/MultiLevelNavigation.js

```diff
@@ -190,6 +190,19 @@ export default class MultiLevelNavigation extends Mutation() {
   }
 
   connectedCallback () {
+    const preventDefaultInputSearch = this.root.querySelectorAll(".preventDefaultInputSearch");
```

Comment:
Please use class names separated by hyphens, such as prevent-default-input-search if possible. Where does this .preventDefaultInputSearch class name come from? I mean, where is the logic to determine whether to add it to the element or not?